export const getAllPatients = async () => {
    try {
        
    } catch (error) {
        
    }
}